provider/cluster_rosa_classic_resource.go

```diff
@@ -373,14 +379,14 @@ func (t *ClusterRosaClassicResourceType) GetSchema(ctx context.Context) (result
 				},
 			},
 			"version": {
```

Comment:
Are you sure?
This will affect both upgrade and cluster create. It will also be a breaking change for any existing users. I chose to leave this one specifically for those reasons and just add "current_version"

Reply:
I think about how to avoid possible mistakes and to make it more clear to the user.
I know that there is description, but still user might be confused.
@arendej any thoughts here?

Reply:
I think we can leave it like that for now, and open a ticket for later 
Also, I agree that it might be a problem and break the current behavior for users that already using "version"